packages/roosterjs-content-model-plugins/lib/tableEdit/editors/features/CellResizer.ts

```diff
@@ -107,24 +107,25 @@ function onDragStart(context: DragAndDropContext, event: MouseEvent): DragAndDro
 
     const { editor, table } = context;
 
-    // Get current selection
-    const selection = editor.getDOMSelection();
-
-    // Select first cell of the table
-    editor.setDOMSelection({
-        type: 'table',
-        firstColumn: 0,
-        firstRow: 0,
-        lastColumn: 0,
-        lastRow: 0,
-        table: table,
-    });
-
-    // Get the table content model
-    const cmTable = getFirstSelectedTable(editor.getContentModelCopy('disconnected'))[0];
-
-    // Restore selection
-    editor.setDOMSelection(selection);
+    // Get Table block in content model
+    let cmTable: ContentModelTable | undefined;
+
+    editor.formatContentModel(
```

Comment:
it seems the 3 calls are the same. Can you extract them to be a shared function?

Just do internal export should be good enough.